mkdocs.yml

```diff
@@ -9,6 +9,7 @@ theme: readthedocs
 nav:
   - Home: index.md
   - Getting started: quickstart.md
+  - Customizing Your Pulp Deployment: customizing.md
```

Comment:
Now this looks like you can first do the quick installation and then customize it. Would it help to call the chapters differently?
* Quick/Easy Installation
* Customized Installation

My goal is to make it obvious that they are seen as alternatives.

Reply:
Yes and no.

The problem is that quickstart.md has instructions (like installing from galaxy) that we lack in customizing.md.

I'd need to reference many sections in it, or copy it.

Sigh, I also just realized that we need to instruct users to specify the hostname, in the `ansible-playbook` command or an inventory file.

Reply:
Then let's put a sentence in quickstart, that it's not for all tshirt sizes and a user should consult customize _while_ performing quickstart.